claat/parser/md/parse.go

```diff
@@ -65,6 +65,13 @@ const (
 	headerFAQ   = "frequently asked questions"
 )
 
+var (
+	importsTagRegexp           = regexp.MustCompile("^<<([^<>()]+)>>\\s*$")
```

Comment:
I'm thinking we could even restrict it to .md file extensions for the moment. Just to be on the safe side.

Reply:
Done restricted for now. If user want remote resource e.g. something in google drive, we can relax this later.